tools/ds-identify

```diff
@@ -1594,21 +1594,42 @@ dscheck_VMware() {
     return "${DS_NOT_FOUND}"
 }
 
-WSL_cloudinit_dir_in() {
+WSL_win2linux_path() {
+    local val=""
+    val="$(wslpath -au "$1" )"
+    _RET="$val"
+}
+
+WSL_linux2win_path() {
+    local val=""
+    val="$(wslpath -am "$1" )"
+    _RET="$val"
+}
+
+WSL_run_cmd() {
+    local val=""
+    val=$(/init "$cmdexe" /c "$@" 2>/dev/null)
+    _RET="$val"
+}
```

Comment:
We have created a number of very simple functions here that increase complexity of the shell code for what otherwise would be a one-liner inline in other call-sites. This complexity also forces call-sites to process changes to the env var _RET which makes those call-sites busier and harder to read for limited benefit. Is the reason we are doing this to get around test_ds_identify test mocking?